src/components/FileReader.vue

```diff
@@ -0,0 +1,68 @@
+<script lang="ts" setup>
+import { ref, reactive, computed } from 'vue'
+import { useImportStore } from '@/stores/import'
+import type MatlabFile from "@/interfaces/IExpectedFileContent"
+
+const docFile: any = ref(null)
+const importFile: any = useImportStore()
+
+const file: any = reactive({})
+const content: any = ref("")
+const loading: any = ref(false)
+
+function readMatlabTxtFile(): void {
```

Comment:
You might want to format your functions a bit better, add some more white-space in sensible places, that would help a ton for just readability (also in the other functions)

Reply:
Done